pkg/customers/documents_test.go

```diff
@@ -239,7 +239,10 @@ func TestDocumentRepository(t *testing.T) {
 			t.Errorf("got %d unexpected documents: %#v", len(docs), docs)
 		}
 		if docs[0].DocumentID != doc.DocumentID {
-			t.Errorf("docs[0].DocumentID=%s doc.DocumentID=%s", docs[0].DocumentID, doc.DocumentID)
+			t.Errorf("docs[0].DocumentID=%q doc.DocumentID=%q", docs[0].DocumentID, doc.DocumentID)
```

Comment:
The `got/want` testing idiom could be useful here:
```go
t.Errorf("document IDs: got: %q, want: %q", docs[0].DocumentID, doc.DocumentID)
```

Reply:
We have this in the logging library that newer projects use.

Reply:
Good call. I'll fix